test/delete.js

```diff
@@ -14,14 +14,20 @@ fastify.register(require('../'), {
   key,
   cookie: {
     path: '/',
-    expires: expires,
+    expires,
     maxAge: 86400
   }
 })
 
 fastify.post('/', (request, reply) => {
+  // using setters
   request.session.set('some', request.body.some)
   request.session.set('some2', request.body.some2)
+
+  // setting natively
+  request.session.set('some3', request.body.some3)
+  request.session.set('some4', request.body.some4)
```

Comment:
what's the difference between the lines above and these?

Reply:
None - good catch - I should be directly setting the props here